Promote/DO App Platform Example (#280)
1Password/scim-examples: README.md

You can deploy 1Password SCIM Bridge on any supported infrastructure that allows ingress by your identity provider and egress to 1Password servers. Here you'll find configuration files and best practices to help with your deployment. Learn more about [automating 1Password provisioning with SCIM](https://support.1password.com/scim/).

## Marketplace apps

> If you'd prefer to deploy 1Password SCIM Bridge using a marketplace, you can use Google Cloud Platform or DigitalOcean:

Reviewer: I just spoke with @ag-adampike on a call and we came up with a slightly modified version of this phrasing, since the context and audience are different from AKB. @black-bryan how does this sound?

```suggestion
For an automated deployment of 1Password SCIM Bridge where some settings are pre-configured for you, use a marketplace app from Google Cloud Platform or DigitalOcean:
```

> ## Automatic (one-click) deployment

Reviewer 2: I would prefer that we standardize on "marketplace apps" for these deployment options:

```suggestion
## Marketplace apps
```

Author: I love this suggestion!

> For a guided deployment, you can use our Google Cloud or DigitalOcean marketplace app to deploy 1Password SCIM Bridge:

Reviewer: As I read this, I wondered what "guided" means, since we have guides (tutorials) for both methods. We should imply that it's easier, which is more accurate:

```suggestion
If you'd prefer to deploy 1Password SCIM Bridge without the command line, you can use the Google Cloud Platform or DigitalOcean marketplace app:
```

I'm open to other suggestions here of course, but I do think that some readers may feel in over their head with all the commands.

Reviewer: I like this phrasing! Slight adjustments to remove "the" and add a comma:

```suggestion
If you'd prefer to deploy 1Password SCIM Bridge using a marketplace, you can use Google Cloud Platform or DigitalOcean:
```

- [Deploy 1Password SCIM Bridge on Google Cloud Platform](https://support.1password.com/scim-deploy-gcp/)
- [Deploy 1Password SCIM Bridge on DigitalOcean](https://support.1password.com/scim-deploy-digitalocean/)

## Custom deployment

> For a more customizabile deployment, whether you choose a container-as-a-service option to allow the infrastructure to be managed by your service provider, or an advanced deployment, you can use these example configurations as a base for a customized deployment.

Reviewer: To simplify this a bit for improved readability:

```suggestion
To customize your deployment, you can use containers as a service or create your own advanced deployment with the examples below as your base.
```

We explain CaaS above so we don't need to again here, similar to how we're not defining what "advanced deployment" means.

### Before you begin

Before you begin deploying 1Password SCIM Bridge, review the [Preparation Guide](/PREPARATION.md). The guide will help you plan for some of the technical components of the deployment and consider some issues you may encounter along the way.

> ### Container-as-a-Service deployment options
> 
> Container-as-a-Service (CaaS) deployments of the 1Password SCIM Bridge will simpify your deployment by utilizing the built-in tools of the CaaS service for DNS and certificate management. This gives you an easy, low cost SCIM bridge with few instrastructure management requirements.

Reviewer: To match the suggestions above and make this a tad more concise. I also changed "few" to "minimal" since that clicks in my mind as "ooo this is going to be nice!"

```suggestion
### Containers as a service deployment options

Containers as a service (CaaS) can simplify your deployment by using the built-in tools of the CaaS for DNS and certificate management. This gives you an easy, low-cost SCIM bridge with minimal infrastructure management requirements.
```

- DigitalOcean App Platform 
  - [Using DigitalOcean Portal](https://support.1password.com/cs/scim-deploy-digitalocean-ap/)
  - [Using DigitalOcean command line tool & the 1Password CLI](/do-app-platform-op-cli) 

### Advanced deployment options

> If you have particular requirements for your environment, an advanced deployment is recommended. These example configurations are just that, examples and can be used for your customized deployment.

Reviewer: To use an active voice here:

```suggestion
If you have particular requirements for your environment, we recommend an advanced deployment. These example configurations will give you a base to create the deployment from, as well as explain what 1Password SCIM Bridge needs to function and how to maintain your bridge once you've deployed it.
```
